utils/src/config.rs

```diff
@@ -102,6 +102,24 @@ pub struct ClusterConfig {
     initial_cluster_state: InitialClusterState,
 }
 
+impl Default for ClusterConfig {
+    #[inline]
+    fn default() -> Self {
+        Self {
+            name: "default".to_owned(),
+            members: HashMap::from([(
+                "default".to_owned(),
+                vec!["http://127.0.0.1:2379".to_owned()],
+            )]),
+            is_leader: false,
+            curp_config: CurpConfig::default(),
+            client_config: ClientConfig::default(),
+            server_timeout: ServerTimeout::default(),
+            initial_cluster_state: InitialClusterState::default(),
+        }
+    }
+}
+
```

Comment:
Please adds a `cfg(test)` for the default if it is used in tests only

Reply:
Config refactoring in the future should be based on this default value, create a default configuration, and then override it based on command line arguments or configuration files, the default value in ClusterConfig represents the startup parameters of a single-node cluster, and the default single-node cluster should be started when xline is run without passing in any parameters